FinniversKit/Sources/Components/SelectionListView/Subviews/SelectionListItemView.swift

```diff
@@ -115,17 +120,20 @@ class SelectionListItemView: UIView {
             textStackView.leadingAnchor.constraint(equalTo: selectionView.trailingAnchor, constant: .spacingM),
             textStackView.bottomAnchor.constraint(equalTo: detailViewsStackView.topAnchor, constant: -.spacingM),
 
-            titleLabel.widthAnchor.constraint(equalTo: descriptionLabel.widthAnchor),
-
-            iconImageView.leadingAnchor.constraint(equalTo: textStackView.trailingAnchor, constant: .spacingM),
-            iconImageView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -.spacingM),
-            iconImageView.centerYAnchor.constraint(equalTo: textStackView.centerYAnchor),
-
             detailViewsStackView.leadingAnchor.constraint(equalTo: selectionView.trailingAnchor, constant: .spacingM),
             detailViewsStackView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -.spacingM),
             detailViewsStackViewBottomConstraint,
         ]
 
+        if case .none = model.icon {} else {
+            contentView.addSubview(iconImageView)
+            constraints.append(contentsOf: [
```

Comment:
Could the icon be placed by using a hierarchy of stackviews? I'm not sure my mental version of the layout is correct here. 😆

Reply:
Looks like it, but I prefer not to touch what's already working here.